openpype/hosts/maya/plugins/create/create_light.py

```diff
@@ -0,0 +1,10 @@
+from openpype.hosts.maya.api import plugin
+
+
+class CreateLight(plugin.Creator):
+    """Light"""
```

Comment:
"Light" seems awfully generic as a description of what this is.

Some things that crossed my mind:

- Is it only ever a single light? or possibly multiple lights?
- Are it always static lights, or also animated lights?
- Would it make more sense as "lights" as opposed to "light"?

Reply:
maybe lightset, lightSet light_set ?